conf/nebula-graphd.conf.production

```diff
@@ -7,6 +7,8 @@
 --enable_optimizer=true
 # Heartbeat interval of communication between meta client and graphd service
 --heartbeat_interval_secs=10
+# Whether to use the configuration obtained from the configuration file
+--local_config=true
```

Comment:
Should the default value be  false?

Reply:
The documentation no longer provides configuration-related command documentation, so the default is to use the configuration file configuration.